### ISSUE TITLE
Staying shadow

### ISSUE THREAD

**Issue by kodmanyagha** [lwouis/alt-tab-macos#1147]
Thanks for your app. This is very good app. Let me explain the bug.

**Bug Description**

Sometimes staying a black shadow after releasing CMD-TAB keys. I can't catch screenshot because it is staying 2 or 3 seconds and it is annoying.

**Steps to reproduce the bug**

1- CMD+TAB key
2- Staying a black shadow about 2-3 seconds.

**OS Specs**

MacBook Pro (16-inch, 2019)
Version 11.6 (20G165)
2,3 GHz 8-Core Intel Core i9
16 GB 2667 MHz DDR4
AMD Radeon Pro 5500M 4 GB

**Comment by lwouis** [comment]
Duplicate of https://github.com/lwouis/alt-tab-macos/issues/563

**Comment by kodmanyagha** [comment]
I think this is different bug because I never get this bug in v6.21.0.

**Comment by lwouis** [comment]
Yes, please see the discussion in that ticket, as well as in the other dozen tickets recently open then closed as duplicates. They are listed at the end of that ticket. It's a recent regression as you noticed.
